src/extension/inpage-script/index.js

```diff
@@ -45,10 +45,12 @@ function registerLightningLinkClickHandler() {
         return;
       }
       const lightningLink = target.closest('[href^="lightning:" i]');
-      const lnurlLink = target.closest('[href^="lnurl" i]');
+      const lnurlLink = target.closest(
+        '[href^="lnurlp:" i],[href^="lnurlw:" i],[href^="lnurlc:" i]'
+      );
       const bitcoinLinkWithLighting = target.closest(
         '[href*="lightning=ln" i]'
-      ); // links with a lightning parameter and a value that starts with ln: payment requests (lnbc...) or lnurl (lnurl*)
+      ); // links with a lightning parameter and a value that starts with ln: payment requests (lnbc...) or lnurl (lnurl[pwc]:)
```

Comment:
There might be an error (unrelated to this PR) in this comment: Shouldn't it be `lightning:` and not `ln:`?

```diff
- // links with a lightning parameter and a value that starts with ln: payment requests (lnbc...) or lnurl (lnurl[pwc]:)
+ // links with a lightning parameter and a value that starts with lightning: payment requests (lnbc...) or lnurl (lnurl[pwc]:)

Reply:
no changes are required there as it indicates links with lightning parameter + value that starts with ln. for eg. lnbc or lnurl